ReactiveCocoa/Swift/Property.swift

```diff
@@ -299,3 +313,33 @@ public func <~ <P: MutablePropertyType>(property: P, producer: SignalProducer<P.
 public func <~ <Destination: MutablePropertyType, Source: PropertyType where Source.Value == Destination.Value>(destinationProperty: Destination, sourceProperty: Source) -> Disposable {
 	return destinationProperty <~ sourceProperty.producer
 }
+
+
+/// Type erased wrapper for composable PropertyType.
+private class _AnyPropertyBoxBase<Value>: PropertyType {
+	var producer: SignalProducer<Value, NoError> {
+		fatalError()
```

Comment:
I don't like this: now we can't guarantee at compile time that this wouldn't crash. Why do you need this case class anyway?